### ISSUE TITLE
Sleep 1m10s Invalid command: m10s

### ISSUE THREAD

**Issue by thingsiplay** [charmbracelet/vhs#287]
```
$ vhs --version
vhs version v0.4.0 (5409c97)
```

I am new to this program and installed it through pacman in Archlinux based distro. I have recorded my session with

```
$ vhs record > demo.tape
```

and when I try to playback it with

```
$ vhs demo.tape
```

it refuses to play with error message:

```
 97 │ Sleep 1m10s
             ^^^^ Invalid command: m10s
```

When I edit the file and change it to `Sleep 1s`, then it plays and creates it back fine.

**Comment by maaslalani** [comment]
Hey there! VHS doesn't support multiple units of time currently (but we probably should in the future) for now instead of `1m10s` please use  `70s`!

**Comment by thingsiplay** [comment]
I understand and everything worked fine after the manual intervention. But vhs was the program that wrote the script and 1m10s.

**Comment by tr3mor** [comment]
This is because recording just writes time.Duration but parser supports only "s" and "ms". The easiest fix is to keep it in sec when it goes above 60,  here is PR #300 .

**Comment by maaslalani** [comment]
This is fixed by #300, thanks again @tr3mor for the fix and @thingsiplay for reporting this issue ❤️
